mscore/magbox.cpp

```diff
@@ -172,12 +172,12 @@ double MagBox::getMag(ScoreView* canvas) const
                   double mag1 = 0;
                   double mag2 = 0;
                   if (MScore::verticalOrientation()) {
-                        mag1 = ch / (ph * 2 * DPI +  MScore::verticalPageGap);
-                        mag2 = cw / (pw * DPI);
+                        mag1 = ch / (ph * 2 +  MScore::verticalPageGap);
+                        mag2 = cw / pw;
                         }
                   else {
-                        mag1 = cw / (pw * 2 * DPI + 50);
-                        mag2 = ch / (ph * DPI);
+                        mag1 = cw / (pw * 2 + 50);
```

Comment:
why 50?

Reply:
I did not change this, so I have no idea.